src/Connectors/test/Connector.Test/PostgreSql/PostgreSqlProviderServiceCollectionExtensionsTest.cs

```diff
@@ -227,4 +231,30 @@ public void AddPostgreSqlConnection_AddsRelationalHealthContributor_WhenCommunit
 
         Assert.NotNull(healthContributor);
     }
+
+    [Fact]
+    public void AddPostgreSqlConnectionWithConnectionStringAndCloudNativeBindingsAddsRightPostgreSqlConnection()
+    {
+        string rootDir = Path.Combine(Environment.CurrentDirectory, "..", "..", "..", "resources", "bindings");
+        Environment.SetEnvironmentVariable("SERVICE_BINDING_ROOT", rootDir);
+        IServiceCollection services = new ServiceCollection();
+        var appsettings = new Dictionary<string, string> {
+            { "postgres:client:ConnectionString", "Server=fake;Database=test;User Id=steeltoe;Password=password;" },
+            { "steeltoe:kubernetes:bindings:enable", "true" }
+        };
+        var configurationBuilder = new ConfigurationBuilder();
+        _ = configurationBuilder.AddInMemoryCollection(appsettings).AddEnvironmentVariables().AddKubernetesServiceBindings();
```

Comment:
I think the test fails if the files aren't found, so I'm a little indifferent on this one, but I'll add it so the expectation is explicit

Reply:
Isn't the test failing when files are missing a good thing? I only mentioned it because in another test, you are passing this parameter, so I assumed you forgot to add it here.

Reply:
I meant that the test would already fail regardless of true/false/nothing here if the files are missing or don't get read correctly... I must have put it in the other one when I was trying to validate what was happening and why I wasn't getting expected test results (side note, the issue was that I hadn't set `Steeltoe:Kubernetes:Bindings:Enabled`=`true`)